examples/gpt2_e2e.py

```diff
@@ -0,0 +1,120 @@
+import os
+import numpy
+from transformers import AutoConfig
+from onnxruntime_customops import mytorch as torch
+
+
+device = 'cpu'
+model_name_or_path = 'gpt2'
+gpt2_core_model_path = './gpt2.onnx'
+gpt2_encoder_model_path = './gpt2_tok.onnx'
+gpt2_decoder_model_path = './gpt2_dec.onnx'
+gpt2_full_model_path = './gpt2_full.onnx'
+
+
+# input_text = ['best hotel in bay area', 'here is an example of gpt2 model']
+input_text = ['best hotel in bay area']
+num_tokens_to_produce = 10
+
+
+def get_cache_directory():
+    cache_dir = os.path.join("../..", "cache_models")
+    if not os.path.exists(cache_dir):
+        os.makedirs(cache_dir)
+    return cache_dir
+
+
+def convert_models():
+    from transformers import GPT2Tokenizer  # noqa
+    from onnxruntime.transformers.gpt2_helper import Gpt2Helper, MyGPT2LMHeadModel  # noqa
+    cache_dir = get_cache_directory()
+
+    tokenizer = GPT2Tokenizer.from_pretrained(model_name_or_path, cache_dir=cache_dir)
```

Comment:
I would move these global variables as argument to the function, it is easier to reuse.

Reply:
This tutorial script, which is logic clear is more important than the style.